UnityProject/Assets/Scripts/Weapons/Gun.cs

```diff
@@ -23,9 +23,9 @@ public class Gun : NetworkBehaviour, IPredictedCheckedInteractable<AimApply>, IC
 	*/
 
 		[SerializeField]
-		private GameObject ammoPrefab = null;
+		protected private GameObject ammoPrefab = null;
```

Comment:
```suggestion
		protected GameObject ammoPrefab = null;
```
Only one access modifier is needed here.